Parse/PFFileUploadController.h

```diff
@@ -0,0 +1,38 @@
+//
+//  PFUploadController.h
+//  Parse
+//
+//  Created by Ken Cooper on 2/20/17.
+//  Copyright © 2017 Parse Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <Bolts/BFTask.h>
+#import "PFFileUploadResult.h"
```

Comment:
Also, we should just use forward declaration through `@class PFFileUploadResult` instead of header import here.